io-engine/src/core/env.rs

```diff
@@ -163,9 +196,15 @@ pub struct MayastorCliArgs {
     #[clap(short = 'T', long = "tgt-iface", env = "NVMF_TGT_IFACE")]
     /// NVMF target interface (ip, mac, name or subnet).
     pub nvmf_tgt_interface: Option<String>,
-    /// NVMF target Command Retry Delay.
-    #[clap(long = "tgt-crdt", env = "NVMF_TGT_CRDT", default_value = "0")]
-    pub nvmf_tgt_crdt: u16,
+    /// NVMF target Command Retry Delay in x100 ms (single integer or three
```

Comment:
Ah interesting, is this not seconds? It's documented as seconds in the helm chart and defaults to 30.

Reply:
It is not seconds, it is x100 ms. In Linux initiator code, it is multiplied by 100; and also I measured it to be sure.